src/components/Common/CrowdsalesList.js

```diff
@@ -29,20 +29,20 @@ export default class CrowdsalesList extends Component {
         <div className="container-fluid">
           <div className="table-row flex-table-header">
             <div className="text">Address</div>
-            <div className="sm-text">Token Name</div>
-            <div className="sm-text">Ticker</div>
-            <div className="sm-text">Date</div>
+            {/*<div className="sm-text">Token Name</div>*/}
```

Comment:
@fernandomg is it temporarily commenting or should we delete these commented lines before merging to master?

Reply:
it's temporary, until we can retrieve the required data for the "Choose contract" popup.

But for the sake of clarity, I can delete it and add it back when required.

Reply:
Oh, I see. It would be double work, if we keep in mind, that we should return it later. Ok, let's leave commented lines here.